src/openvic-simulation/Modifier.cpp

```diff
@@ -138,4 +161,4 @@ namespace OpenVic { //so the compiler shuts up
 		}
 		return stream;
 	}
-}
\ No newline at end of file
+}
```

Comment:
Advise minor reduction in nesting if possible.

Reply:
This function is going to be rewritten to use a better system than `default_callback`.